airbyte-cdk/python/airbyte_cdk/sources/file_based/stream/default_file_based_stream.py

```diff
@@ -243,6 +252,9 @@ async def _infer_schema(self, files: List[RemoteFile]) -> Mapping[str, Any]:
             # number of concurrent tasks drops below the number allowed.
             done, pending_tasks = await asyncio.wait(pending_tasks, return_when=asyncio.FIRST_COMPLETED)
             for task in done:
+                if task.exception():
```

Comment:
Exceptions were not caught as part of this loop. This might be a breaking change though as before, if one file was not being parsed, it was fine as we could rely on the other files.

Are we ok with this breaking change?